### DISCUSSION TITLE
Handling of existing filenames with dots

### DISCUSSION THREAD

**Discussion by schmunk42** [requarks/wiki#5863] [Error / Bug Report] [answered]
> `Wiki.js 2.5.291` with PgSQL

I have imported our existing markdown wiki via git.

We have several pages in the form `subdomain.domain.tld.md` documenting a website.

The content is available, but when editing and saving a page it creates a new page `subdomaindomaintld.md`.

When creating a completely new page with dots in the filename like above the path is getting filled with `subdomain.domain` which is only marked as invalid, when trying to save the page.

Could `.` (dots) simply be allowed?
Or it might be an option to completely disallow editing of files with a dot in the name.

**Comment by NGPixel** [comment, marked as answer]
No plans on allowing dots in paths, as they are used to separate levels.

**Reply by schmunk42** [reply]
Which levels do you mean? I thought slashes `/` would be used for that.

**Reply by NGPixel** [reply]
Internally, dots are also used in path indexing.

**Reply by schmunk42** [reply]
I think *Git > Import Everything* should handle that in "some way" - it might just ignore pages with "Illegal Character", defined here  https://docs.requarks.io/guide/pages and print a warning/error.

Because in the current setup you might really mess up your repo like mentioned above (new pages are created).
